hypervisor/include/dm/vpci.h

```diff
@@ -32,7 +32,10 @@
 
 #include <spinlock.h>
 #include <pci.h>
+#include <list.h>
 
+#define VDEV_LIST_HASHBITS 6U
```

Comment:
Even 2^^6 = 64 is too large for a UOS. I thought a UOS  normally has ~10 vdevs. In this case, 8 is good enough. SOS may have ~100 vdevs, this case, 64 is the right value.  Balance both of them, let us choose 16. 

For PDEV side, it is similar, let us use 64.